test/rules/best-practises/no-unused-import.js

```diff
@@ -27,6 +27,26 @@ describe('Linter - no-unused-import', () => {
     assertErrorMessage(report, 'imported name A is not used')
   })
 
+  it('should not crash on, but also not recognize, malformed inheritdoc statements', () => {
```

Comment:
These behaviours (not crashing and not recognising) aren't coupled. Maybe this test should be two tests.